browse-scaler/src/__test__/resize.test.js

```diff
@@ -1,55 +1,61 @@
-const fs = require('fs');
-const sharp = require('sharp');
-const { resizeImage, notFound } = require('../resize');
+const fs = require("fs");
+const sharp = require("sharp");
+const { resizeImage, notFound } = require("../resize");
 
-describe('Image resizing', () => {
-  test('Images are resized as expected', async () => {
-    const image = fs.readFileSync('stars.jpg');
+describe("Image resizing", () => {
+  test("Images are resized as expected", async () => {
+    const image = fs.readFileSync("stars.jpg");
```

Comment:
when doing `readFileSync` need to include the full path to the file. This was one of the errors we were saying yesterday. Change "stars.jpg" to `__dirname + "/stars.jpg"`.